fxa-auth-db-server/test/backend/db_tests.js

```diff
@@ -2180,11 +2180,22 @@ module.exports = function(config, DB) {
                     .then(function () {
                       testAccount.email = testSecondaryEmail.email
                       testAccount.normalizedEmail = testSecondaryEmail.normalizedEmail
+
+                      // Create new account with secondary email that was deleted
                       return db.createAccount(testAccount.uid, testAccount)
                         .catch(t.fail)
                     })
                     .then(function (res) {
                       t.deepEqual(res, {}, 'successfully created an account')
+
+                      // Attempt to create secondary email address
+                      return db.createEmail(testAccount.uid, testSecondaryEmail)
+                        .then(() => {
+                          t.fail(new Error('Should not have created secondary email'))
```

Comment:
Ah yes so we need this test, but we also need a test to capture this issue we found:

> Create account B (b@b.com), this was previously failing, but now works

The test we need to capture 

* Create account A (a@a.com)
* Add secondary email (b@b.com) to account A
* Delete account A (a@a.com)
* Create account C (c@c.com)
* *Add secondary email (b@b.com) to account C*

Before the last step would fail I think